Composer/packages/ui-plugins/lg/src/LgField.tsx

```diff
@@ -35,11 +35,20 @@ const getInitialTemplate = (fieldName: string, formData?: string): string => {
 };
 
 const LgField: React.FC<FieldProps<string>> = props => {
-  const { label, id, description, value, name, uiOptions } = props;
+  const { label, id, description, value, name, uiOptions, schema } = props;
```

Comment:
Hi @tdurnford can you please help me take a look at here, I trying to get the `InputType` from `schema` property, (just like `LuisIntentEditor.tsx` did ), but data value passed to here seems different.

Reply:
@zhixzhan  The `LgField` only has access to the schema for the particular field it is overriding - `IActivityTemplate` - while the `LuisIntentEditor` has access to the whole schema and can pull the `$kind` from it. Even though field overrides and intent editors behave differently, it looks like you can still get the `$kind` through the `useShellApi`.

```javascript
const { data } = useShellApi();
const kind = data?.$kind;
```

Reply:
@tdurnford get it, thanks!